tests/PhingTesterIntegrationTest.php

```diff
@@ -15,7 +16,7 @@ public function testExecuteTarget(): void
 		$target = __FUNCTION__;
 		$tester->executeTarget($target);
 
-		$this->assertTrue(true); // build should not fail and reach this
+		Assert::assertTrue(true); // build should not fail and reach this
```

Comment:
This should rather be replaced with `expectNotToPerformAssertions()` :)

Reply:
ok, replaced :)